roles/common/tasks/main.yml

```diff
@@ -46,19 +46,19 @@
   apt: name=psmisc
 - name: Install aptitude
   apt: name=aptitude
-- name: Upgrade APT to the lastest packages
-  apt: upgrade=full
-- name: Rebooting machine so new kernel is active
-  become: yes
-  shell: sleep 5 && shutdown -r now removes=/var/run/reboot-required
-  async: 1
-  poll: 0
-  ignore_errors: true
-  register: restarted
-- name: Waiting for reboot...
-  become: false
-  local_action: wait_for host="{{ inventory_hostname }}" search_regex=OpenSSH port=22 delay=90 timeout=120 state=started
-  when: restarted.changed
+  #- name: Upgrade APT to the latest packages
```

Comment:
Why did you comment this out? Did it not work for you?

Reply:
These were failing. As I am using the latest version of Ubuntu, it is not a big issue for the moment. We should create an issue and check what is causing this.